src/lessons/02/Lecturer.ts

```diff
@@ -0,0 +1,41 @@
+import { Contact, Course, Position } from '@type/lesson02'
+
+export class Lecturer {
+  constructor(
+    private readonly _name: string,
+    private readonly _surname: string,
+    private readonly _position: Position,
+    private readonly _company: string,
+    private readonly _experience: number,
+    private readonly _courses: Course[],
+    private readonly _contacts: Contact[],
+  ) {}
+
+  get name(): string {
+    return this._name
```

Comment:
not critical, but sometime you forget about semicolon) Maybe it's a good time to configure `prettier `for it)